packages/terra-clinical-item-display/docs/README.md

```diff
@@ -18,13 +18,13 @@ import ItemDisplay from 'terra-clinical-item-display';
   text="display text"
   textStyle="attention"
   isTruncated
-  icon=<img alt="Graphic" />
+  isInvalid
```

Comment:
Lets update these examples to reflect a 'truer' implementation where `isInvalid={this.state.isValid}`

Reply:
Just clarifying.. Are you suggesting using `this.state.isValid` which would be defined by a consumer in our example? The current example indicates this is a bool with no additional cognitive load of determining what state means.